app/models/EmploymentExpensesRequestModel.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import play.api.libs.json._
+
+sealed trait EmploymentExpensesRequestModel
+
+final case class ExpenseType(
+                        businessTravelCosts: BigDecimal,
+                        jobExpenses: BigDecimal,
+                        flatRateJobExpenses: BigDecimal,
+                        professionalSubscriptions: BigDecimal,
+                        hotelAndMealExpenses: BigDecimal,
+                        otherAndCapitalAllowances: BigDecimal,
+                        vehicleExpenses: BigDecimal,
+                        mileageAllowanceRelief: BigDecimal
```

Comment:
can reuse the existing ExpensesType case class here as all fields should be optional

Reply:
Done